esp/src/src-react/components/LandingZone.tsx

```diff
@@ -93,33 +82,167 @@ export const LandingZone: React.FunctionComponent<LandingZoneProps> = ({
     const [showDropZone, setShowDropzone] = React.useState(false);
     const [uploadFiles, setUploadFiles] = React.useState([]);
     const [showFileUpload, setShowFileUpload] = React.useState(false);
-    const [targetDropzones, setTargetDropzones] = React.useState([]);
+    const [layout, setLayout] = React.useState<object>();
+    const [dockpanel, setDockpanel] = React.useState<ResetableDockPanel>();
+    const [dropzones, setDropzones] = React.useState<WsTopology.TpDropZone[]>([]);
+    const [selectedDropzone, setSelectedDropzone] = React.useState<WsTopology.TpDropZone>();
+
+    const [data, setData] = React.useState<any[]>([]);
+    const {
+        selection, setSelection,
+        setTotal,
+        refreshTable } = useFluentStoreState({});
+
+    const [userAddedFiles, setUserAddedFiles] = React.useState<any[]>([]);
+    const [lzPath, setLzPath] = React.useState<string>("");
+    const [pathSep, setPathSep] = React.useState<string>("/");
+
+    const [treeItems, setTreeItems] = React.useState<FlatItem[]>([]);
+    const [openItems, setOpenItems] = React.useState<Iterable<TreeItemValue>>([]);
+
+    const store = useConst(() => userKeyValStore());
+
+    React.useEffect(() => {
+        if (dockpanel) {
+            //  Should only happen once on startup  ---
+            store.get("LzLayout").then(value => {
+                if (!value || value === "undefined") {
+                    const layout: any = dockpanel.layout();
+                    if (Array.isArray(layout?.main?.sizes) && layout.main.sizes.length === 2) {
+                        layout.main.sizes = [0.2, 0.8];
+                        dockpanel.layout(layout).lazyRender();
+                        setLayout(layout);
+                        store?.set("LzLayout", JSON.stringify(layout), true);
+                    }
+                } else {
+                    const layout = JSON.parse(value);
+                    dockpanel.layout(layout);
+                }
```

Comment:
This logic should be commoned up with Metrics and ECLArchive (inside a hook?) - can you open a ticket for this and add the remark:  
```
Don't check value, do the JSON.parse in a try block and test the result for validity.
Add a version to the serialized string and reset when they don't match.
```